packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -10,27 +22,32 @@
     self-stretch
     border-none
     bg-transparent;
-  color: theme("borderColor.color.input");
+
   padding-block: theme("spacing.3");
   padding-inline: theme("spacing.1");
   line-height: 0;
+  cursor: move;
+
+  background-color: var(--calcite-handle-background-color);
+  color: var(--calcite-handle-text-color);
 
   calcite-icon {
     color: inherit;
   }
 }
-
-:host(:not([disabled])) .handle {
-  @apply cursor-move;
-  &:hover {
-    @apply bg-foreground-2 text-color-1;
-  }
-  &:focus {
-    @apply text-color-1 focus-inset;
-  }
-  &--selected {
-    @apply bg-foreground-3 text-color-1;
-  }
+:host(:not([disabled]):hover) {
+  --calcite-handle-text-color: var(--calcite-color-text-1);
```

Comment:
@Elijbet how can a user set the handle color when hovered? I think we need separate vars for these.

Reply:
@driskull @alisonailea Ali I changed the previous setup to incorporate separate states for hover, focus, and selected states.

Reply:
We decided to only add state tokens for component "parts" (e.g., close button on alert), but not for the host itself (see https://github.com/Esri/calcite-design-system/pull/8645#discussion_r1488300762).

Reply:
That means if a user wants to customize the hover text color on this component they would need to look into the source code because they wouldn't realize they need this selector: `:host(:not([disabled]):hover) {`. In a lot of cases its not just as simple as `host(:hover){`. IMO it would be much simpler for the end user with a separate component token and they wouldn't have to be aware of these implementation specifics.